### DISCUSSION TITLE
How does autogluon calculate 'the weight' of weighted_ensemble?

### DISCUSSION THREAD

**Discussion by Tobi3238** [autogluon/autogluon#3529] [Q&A] [answered]
Hi, 

How does autogluon calculate 'the weight' of weighted_ensemble?
How does autogluon determine 'model_weight'?
(ex) this is generated from .get_info()
'name': 'WeightedEnsemble_L3'
'model_weights': {'CatBoost_BAG_L2': 0.8333333333333334, 'XGBoost_BAG_L2': 0.16666666666666666}}}}

Best,

**Comment by shchur** [comment, marked as answer]
Hi, the WeightedEnsemble algorithm is implemented in https://github.com/autogluon/autogluon/blob/master/core/src/autogluon/core/models/greedy_ensemble/ensemble_selection.py#L29, this method is based on the paper https://www.cs.cornell.edu/~caruana/ctp/ct.papers/caruana.icml04.icdm06long.pdf.

**Reply by Tobi3238** [reply]
Thanks !
